next/components/Network/components/Datasets/components/MainColumn/projectsTable.tsx

```diff
@@ -0,0 +1,26 @@
+import { FluidPaper } from "@clevercanary/data-explorer-ui/lib/components/common/Paper/paper.styles";
+import React from "react";
+import { ProjectsEntityResponse } from "../../../../../../apis/azul/hca-dcp/common/entities";
+import { getProjectsTableColumns } from "../../../../../../viewModelBuilders/viewModelBuilders";
+import { DetailViewTable } from "../../../../../common/Table/table.styles";
+
+interface ProjectsTableProps {
+  networkPath: string;
+  projects: ProjectsEntityResponse[];
+}
+
+export const ProjectsTable = ({
+  networkPath,
+  projects,
+}: ProjectsTableProps): JSX.Element => {
+  return (
+    <DetailViewTable
+      columns={getProjectsTableColumns(networkPath) as any}
```

Comment:
I'm casting this to `any` just so typescript stop complaining about some missing property. Needs more investigation to remove the cast.

Reply:
The casting can be resolved by adding:

`as typeof DXDetailViewTable` to https://github.com/clevercanary/hca-data-portal/blob/4288d45d4331c95912550e0b43b4ca7c7038f66f/next/components/common/Table/table.styles.ts to the DetailViewTable styled component.